### PR TITLE
Master fixcanarias jco

### PR DESCRIPTION
**Pull request by jco-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/data/res_country_group.xml
Comment (rugo-odoo) [inline addons/account/data/res_country_group.xml:1]: Indentation is not 4 spaces and missing empty line at end of file.

File: addons/account/views/res_country_group_view.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="inherits_website_sale_country_group_form" model="ir.ui.view">
```
Comment (rugo-odoo) [inline addons/account/views/res_country_group_view.xml:4]: Why `website_sale` ???

File: addons/l10n_es/data/res_country_group.xml
Comment (rugo-odoo) [inline addons/l10n_es/data/res_country_group.xml:1]: Indentation is not 4 spaces and missing empty line at end of file.

File: addons/l10n_es/data/template/account.fiscal.position-es_common.csv [hunk 2]
```diff
@@ -1,816 +1,816 @@
...
-"","","","","","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_iva4_bc"
-"","","","","","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_irpfnrnue0p"
-"","","","","","","","","","account_tax_template_p_iva4_sc","account_tax_template_p_iva4_sc"
-"","","","","","","","","","account_tax_template_p_iva4_sc","account_tax_template_p_irpfnrnue0p"
-"","","","","","","","","","account_tax_template_s_iva0","account_tax_template_s_iva0"
-"","","","","","","","","","account_tax_template_s_iva0","account_tax_template_s_irpfnrnue0"
-"","","","","","","","","","account_tax_template_p_iva0_bc","account_tax_template_p_iva0_bc"
-"","","","","","","","","","account_tax_template_p_iva0_bc","account_tax_template_p_irpfnrnue0p"
\ No newline at end of file
+"id","sequence","name","name@es","name@ca","auto_apply","vat_required","country_id","country_group_id","tax_ids/tax_src_id","tax_ids/tax_dest_id","account_ids/account_src_id","account_ids/account_dest_id","state_ids"
```
Comment (rugo-odoo) [inline addons/l10n_es/data/template/account.fiscal.position-es_common.csv:1]: If I'm not mistake, we don't do anything with `state_ids` here. Why are we adding it ?

File: addons/l10n_nl/data/res_country_group.xml
Comment (rugo-odoo) [inline addons/l10n_nl/data/res_country_group.xml:1]: Indentation is not 4 spaces and missing empty line at end of file.

File: addons/l10n_es/data/res_country_group.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+	<record id="mainland_spain" model="res.country.group">
```
Comment (rugo-odoo) [inline addons/l10n_es/data/res_country_group.xml:3]: We do `mainland_nl` for the Netherlands, let's do `mainlaind_es` here to be consistent (?)

**Review by rugo-odoo — CHANGES_REQUESTED** [review]

**Comment by jco-odoo** [comment]
@rugo-odoo Should be ready

**Comment by jco-odoo** [comment]
@rugo-odoo rebased

**Comment by rugo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**